pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.towncrier]
+    package = "cocotb"
+    directory = "documentation/source/newsfragments"
+    filename = "documentation/source/release_notes.rst"
+    issue_format = ":pr:`{issue}`"
```

Comment:
Is this for issues and PRs? (It doesn't matter much in GitHub, but would still be good to have the right semantics.)

Reply:
In the README, I now leave it to the author to pick issue or PR number, but we have to artificially pick one way here. It will end up as a working link always.